Packages/com.unity.inputsystem/InputSystem/Plugins/InputForUI/InputSystemProvider.cs

```diff
@@ -16,7 +16,9 @@ internal class InputSystemProvider : IEventProviderImpl
 
         Configuration m_Cfg;
 
+        public InputActionAsset inputActionAsset => m_InputActionAsset;
```

Comment:
Why did this change to public visibility?

Reply:
Updated in e99ed66a5b8bc1887f3527cf35d03b809e34d548 to be more restrictive.